src/framework/ui/overflowMenu/overflowMenu.component.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {
+  ViewProps,
+  View,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+  styled,
+} from '@kitten/theme';
+import {
+  OverflowMenuItemType,
+  OverflowMenuItem as OverflowMenuItemComponent,
+  Props as OverflowMenuItemProps,
+} from './overflowMenuItem.component';
+import {
+  Popover as PopoverComponent,
+  Props as PopoverProps,
+} from '../popover/popover.component';
+import { Omit } from '../service/type';
+
+type MenuItemElement = React.ReactElement<OverflowMenuItemProps>;
+export type MenuItemType = Omit<OverflowMenuItemType, 'size'>;
+
+interface OverflowMenuProps {
+  children: React.ReactElement<any>;
+  items: MenuItemType[];
+  size?: string;
+  onSelect?: (index: number) => void;
+}
+
+const Popover = styled<PopoverComponent, PopoverProps>(PopoverComponent);
+const OverflowMenuItem =
+  styled<OverflowMenuItemComponent, OverflowMenuItemProps>(OverflowMenuItemComponent);
+
+export type Props = & StyledComponentProps & OverflowMenuProps & Omit<PopoverProps, 'content'>;
+
+export class OverflowMenu extends React.Component<Props> {
+
+  static defaultProps: Partial<Props> = {
```

Comment:
can be omitted if is marked as default in mappings

Reply:
Done